ACS Publications.js

```diff
@@ -180,29 +180,15 @@ function doWeb(doc, url){
 }
 
 function scrape(items, opts){
-	//get citation export page's source code;
 	for(var i=0, n=items.length; i<n; i++) {
-		(function(item) {
-			var url = '/action/showCitFormats?doi=' + encodeURIComponent(item.doi);
-			//Z.debug(url);
-			ZU.doGet(url, function(text){
-				//Z.debug(text)
-				//get the exported RIS file name;
-				var downloadFileName = text.match(
-					/name=\"downloadFileName\" value=\"([A-Za-z0-9_\-\.]+)\"/)[1];
-				Zotero.debug("downloadfilename= "+downloadFileName);
-				processCallback(item, opts, downloadFileName);
-			});
-		})(items[i]);
+		processCallback(items[i], opts);
 	}
 }
 
 function processCallback(fetchItem, opts, downloadFileName) {
 		var baseurl = "/action/downloadCitation";
 		var doi = fetchItem.doi;
-		var post = "doi=" + encodeURIComponent(doi) + "&downloadFileName=" + encodeURIComponent(downloadFileName)
-			+ "&include=abs&format=refman&direct=on"
-			+ "&submit=Download+article+citation+data";
+		var post = "//pubs.acs.org/action/downloadCitation?direct=true&doi="+encodeURIComponent(fetchItem.doi)+"&format=ris&include=abs&submit=Download+Citation"
```

Comment:
Hm... maybe it is not anymore that a good idea: http://www.paulirish.com/2010/the-protocol-relative-url/